comp/core/workloadmeta/def/types.go

```diff
@@ -678,6 +678,7 @@ type KubernetesPod struct {
 	IP                         string
 	PriorityClass              string
 	QOSClass                   string
+	GPUType                    string
```

Comment:
this can be a slice of string
```suggestion
	GPUType                    []string
```